tsl/test/sql/tsl_tables.sql

```diff
@@ -14,7 +14,7 @@ RETURNS VOID
 AS :TSL_MODULE_PATHNAME, 'ts_test_bgw_job_delete_by_id'
 LANGUAGE C VOLATILE STRICT;
 
-\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
+SET ROLE :ROLE_DEFAULT_PERM_USER;
```

Comment:
It is difficult to parse the .sql file (often needed while debugging a regression test) and figure out which roles are getting reset. Keeping this as a specific user makes the role-user explicit. Could you undo these changes and retain the explicit \c <dbname> <user> syntax?

Reply:
I see what you are saying, and I can undo for this PR.

Still, I do think creating new sessions throughout tests should be avoided for performance reasons but also debugging. It's not possible to run tests through a debugger if you create new sessions all the time. I guess one alternative is to combine the `RESET ROLE` with `SELECT current_user;`?

Reply:
There are 2 cases where we use different users as part of the test  (1) Some of the tests require testing with different users - primarily for grants and permission testing. ( 2) With the current testing framework, we also do all the setup work (that requires super user permissions) as part of the tests. Resetting roles + show current_user should work in this scenario.

Reply:
Actually, let me revert this and will adopt a consistent approach in separate PR.